util.sh

```diff
@@ -241,3 +241,14 @@ function manta_update_env {
 
     echo "alias js2json='node -e '\''s=\"\"; process.stdin.resume(); process.stdin.on(\"data\",function(c){s+=c}); process.stdin.on(\"end\",function(){o=eval(\"(\"+s+\")\");console.log(JSON.stringify(o)); });'\'''" >> /root/.bashrc
 }
+
+#
+# manta_setup_boray_schemas: run the schema-manager to define the boray
+# schemas on the associated shards if they do not yet exist.
+#
+function manta_setup_boray_schemas {
+    if [[ -x /opt/smartdc/boray/bin/schema-manager ]] ; then
+        echo "Setting up boray schemas"
+        /opt/smartdc/boray/bin/schema-manager || fatal "unable to set up boray schemas"
+    fi
```

Comment:
If the file is not found or not executable do we want to print or return an error here?

Reply:
I think `fatal` is fine.  But if we call the function and the `/opt/smartdc/boray/bin/schema-manager` doesn't exist, it will just silently return.  Is that what we want?